public/template.json

```diff
@@ -583,6 +583,7 @@
           "value": ""
         },
         "favorite": false,
+        "gmNotes": "",
```

Comment:
I believe we already have a `secretDescription` field that we can maybe use @Pjb518 ?

Reply:
Correct. It's defined in the base item template. The key is `system.secretDescription`.